src/extras.py

```diff
@@ -58,7 +58,12 @@ def augment_with_salary(record: Record) -> str:
 def _get_oo_id_mapping() -> Mapping[str, str]:
     if OO_CACHE.currsize == 0:
         log.info("Refreshing OO ID cache")
-        response = requests.get(OO_ID_SOURCE_URL)
+        try:
+            response = requests.get(OO_ID_SOURCE_URL, timeout=60)
+            response.raise_for_status()
+        except Exception as err:
```

Comment:
`requests.exceptions.RequestException` is the base exception in requests.
`requests.exceptions.Timeout` is specific to a timeout and might be useful to catch for a retry.

Reply:
That's really good to know! I'll change it to the base one